packages/push/src/PushRegistrationInterface.ts

```diff
@@ -0,0 +1,17 @@
+export interface PushRegistrationOptions {
+  alias?: string;
+  categories?: string[];
+  timeout?: number;
+}
+
+export interface PushRegistrationWebpushOptions extends PushRegistrationOptions {
+  serviceWorker?: string;
+}
+
+/**
+ * Interface for PushRegistration objects.
+ */
+export interface PushRegistrationInterface {
```

Comment:
I know what you did here, but to be very honest for this case I think we don't need a `PushRegistrationInterface`